packages/next/src/server/async-storage/request-async-storage-wrapper.ts

```diff
@@ -45,10 +47,21 @@ function getMutableCookies(
   return MutableRequestCookiesAdapter.wrap(cookies, onUpdateCookies)
 }
 
+export type WrapperRenderOpts = RenderOpts &
```

Comment:
This will only pick `after` otherwise it will merge with `RenderOpts.experimental`.

```suggestion
export type WrapperRenderOpts = Omit<RenderOpts, 'experimental'> &
```

After this you can remove the `FIXME` and `as` cast in `adpater.ts`

Reply:
fixed, nice!